CHANGELOG.md

```diff
@@ -2,13 +2,18 @@
 
 ## 26
 
-* Added a configuration option under `system.io` to throttle the number of I/O queries that Cromwell makes.
+* Added a configuration option under `system.io` to throttle the number of I/O queries that Cromwell makes, as well as configure retry parameters.
  This is mostly useful for the JES backend and should be updated to match the GCS quota available for the project.
  
 ```
 system.io {
+  # Global Throttling - This is mostly useful for GCS and can be adjusted to match
+  # the quota availble on the GCS API
   number-of-requests = 100000
   per = 100 seconds
+  
+  # Number of times an I/O operation should be attempted before giving up and failing it.
+  nb-attempts = 5
```

Comment:
can this be something more descriptive? e.g. `number-of-attempts`. I don't think it needs to have `io` in the name as that's implied by the structure but `nb` caused me to say "huh?"